libeos-payg/util.c

```diff
@@ -31,24 +31,28 @@ static gboolean payg_legacy_mode = FALSE;
 /* Force a poweroff in situations where we are not able to enforce PAYG. This
  * is intended to be used as a GSourceFunc, e.g. with g_timeout_add_seconds()
```

Comment:
Maybe expand the documentation to mention that if `user_data` is supplied, it must be a pointer to the source ID to be cleared.

Reply:
Good point.